src/simulation/deviceInterface/stepperMotor/stepperMotor.h

```diff
@@ -0,0 +1,93 @@
+/*
+ ISC License
+
+ Copyright (c) 2025, Laboratory for Atmospheric and Space Physics, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef _STEPPERMOTOR_
+#define _STEPPERMOTOR_
+
+#include "architecture/_GeneralModuleFiles/sys_model.h"
+#include "architecture/messaging/messaging.h"
+#include "architecture/msgPayloadDefC/MotorStepCommandMsgPayload.h"
+#include "architecture/msgPayloadDefC/StepperMotorMsgPayload.h"
+#include <stdint.h>
+
+/*! @brief Stepper motor class. */
+class StepperMotor : public SysModel {
+   public:
+    StepperMotor() = default;   //!< Constructor
+    ~StepperMotor() = default;  //!< Destructor
+
+    void Reset(uint64_t currentSimNanos) override;        //!< Reset member function
```

Comment:
The methods have doxygen documentation strings in both the *.h and *.cpp files.  This caused the *.rst file to have repeated documentation text.  Plus, it creates two places to update documentation in the future. A [squash] labeled commit was pushed which corrects this.